index.html

```diff
@@ -158,8 +158,15 @@
            observer.disconnect();
       }
     });
-    // subscribe to Resource-Timing and User-Timing events
-    observer.observe({entryTypes: ['resource', 'mark', 'measure']});
+    observer.observe({
+      // subscribe to Resource-Timing, User-Timing, and Server-Timing events
+      entryTypes:
+        ['resource', 'mark', 'measure', 'server'],
+
+      // retrieve the buffered Resource-Timing, User-Timing, and Server-Timing events
+      bufferedTypes:
+        ['resource', 'mark', 'measure', 'server']
```

Comment:
I think we need to decide here. If a PO registers late (minutes after `onload`) with `buffered: true`:

1) retrieve all entries that were recorded up until that time
-this means we need to buffer all entries during the entire lifetime of the page, until limit
-entries returned to the PO would be exactly the same as those returned by `window.performance.getEntries()`, presuming we haven't reached our limit

2) retrieve all entries that were recorded up until the `load` event
-this will require a change for Resource and User Timing
-devs who poll on `getEntries` after `onload` will never get anything new

Fairly certain we want 2), PR coming...